src/main/java/org/eulu/bookshop/model/Role.java

```diff
@@ -0,0 +1,30 @@
+package org.eulu.bookshop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "roles")
+public class Role {
```

Comment:
Have you checked how your project works after the changes?)
```suggestion
public class Role implements GrantedAuthority {
```